extensions/terminal-suggest/src/completions/code-insiders.ts

```diff
@@ -2,12 +2,55 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import code from './code';
+import { filepaths } from '../helpers/filepaths';
+import code, { parseInstalledExtensions } from './code';
 
 const codeInsidersCompletionSpec: Fig.Spec = {
 	...code,
 	name: 'code-insiders',
 	description: 'Visual Studio Code Insiders',
+	options: [
+		...(code as Fig.Subcommand).options!,
+		{
+			name: '--disable-extension',
+			description: 'Disable an extension',
+			args: {
```

Comment:
We don't want to duplicate the options here, if they require the CLI then we can export a function that creates the generator functions `createCodeGenerators(cliName: string)`